.azure-pipelines/templates/test-all-checks.yml

```diff
@@ -481,6 +481,9 @@ jobs:
     - checkName: vsphere
       displayName: vSphere
       os: linux
+    - checkName: weblogic
+      displayName: Weblogic
```

Comment:
Looks like the name is WebLogic https://en.wikipedia.org/wiki/Oracle_WebLogic_Server